docs/app/pages/components/components-sections/select/select/select.component.html

```diff
@@ -206,6 +206,22 @@
     <tr uxd-api-property name="placeholder" type="string">
         The placeholder text which appears in the text input area when it is empty.
     </tr>
+    <tr uxd-api-property name="tagTemplate" type="TemplateRef">
+        <p>A template which will be rendered as the content of each tag. The following context
+            properties are available in the template:</p>
```

Comment:
I understand that this is lifted from the Tag Input documentation, but I think it should be amended to be more relevant to the select component. The "tag" in this case is a selected option in multiple select mode, so therefore "selected option" may be a better substitution for "tag" throughout this documentation entry.